partcad/src/partcad/actions/part_actions.py

```
@@ -0,0 +1,142 @@
+import os
```

Ruff (0.8.2)

`os` imported but unused

Remove unused import: `os`

(F401)